tools/mpremote/mpremote/transport.py

```diff
@@ -63,6 +64,8 @@ def _convert_filesystem_error(e, info):
         return FileExistsError(info)
     if "OSError" in e.error_output and "ENODEV" in e.error_output:
         return FileNotFoundError(info)
+    if "OSError" in e.error_output and "EINVAL" in e.error_output:
+        return OSError(errno.EINVAL, info)
```

Comment:
Is this needed because trying to remove a mount point returns EINVAL?

Probably the VFS code should be fixed so attempting to do `os.remove('/')` raises EPERM instead of EINVAL, but that's out of scope for this PR.  So I guess catching EINVAL is OK for now.

Reply:
> trying to remove a mount point returns EINVAL

Indeed, that is through observation on 5 different ports